src/components/reference/_comment-triggers.mdx

```diff
@@ -0,0 +1,5 @@
+| Type of scan | Product name | Trigger condition | How to set up |
+| --  | -- |  -- | --- |
+| Static application security testing (SAST) | Semgrep Code | A PR comment appears when a finding is generated by a rule in **Comment or Block mode**. This means you can fully customize what comments your developers receive. | Complete the steps in the following sections: <ol><li>[Confirm your Semgrep account's connection and access to your source code manager](#confirm-account-connection-and-access).</li><li>[Configure PR comments for Semgrep Code](#configure-pr-comments-for-semgrep-code).</li></ol> |
```

Comment:
I think this section is missing on the GitLab page (it's present on the GitHub page)

Reply:
Thanks for this catch. Added another component.